addons/move-to-top-bottom/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Move costume to top or bottom",
-  "description": "Adds a right click context menu item to move a costume or a sound to the top or the bottom of the list.",
+  "description": "Adds buttons to the right click context menu for costumes and sounds to move them to the top or the bottom of the list.",
```

Comment:
which do we use more? "right click context menu" or "context menu"?

Reply:
I see [1](https://github.com/ScratchAddons/ScratchAddons/blob/4d70ad6f65f32c47585a29060b6d20bc4d2837e3/addons/hide-delete-button/addon.json#L3), [2](https://github.com/ScratchAddons/ScratchAddons/blob/a3c035b087962391eef6747fc4d91c9ae2d6025e/addons/editor-colored-context-menus/addon.json#L3) other uses of "right click context menu", and I don't see a use of "context menu" alone. IMO, we can leave this as it is, and changing it might work best in a new PR.